src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
The use of `box-sizing: border-box;` is good, but it should be applied to all elements using the universal selector `*` to ensure consistent box-sizing across the entire document. Consider adding `* { box-sizing: border-box; }` at the beginning of your CSS.